scripts/validate-test-results

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env perl
+
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use strict;
+use warnings;
+
+use Mojo::JSON qw(decode_json);
+use Mojo::Path;
+use Mojo::File qw(path);
+
+my @args = @ARGV;
+my @test_results_to_check = ();
+
+while(@args){
+	my $arg = shift @args;
+	@args = (@args, glob("$arg/result-*.json")) if -d $arg;
+	next unless $arg =~ m/result-.+\.json$/;
+	push @test_results_to_check, $arg if -f $arg;
+}
+
+while(@test_results_to_check){
```

Comment:
```suggestion
while(@test_results_to_check) {
```

But this likely shouldn't be a `while`-loop (with a `shift` operation inside). It would be much simpler to avoid modifying the array and just iterating though it via a `for`-loop.

Reply:
Resolving in favor of @perlpunk 's script.